packages/koehnlab/spin_hamiltonians/spin_systems.py

```diff
@@ -0,0 +1,333 @@
+import numpy as np 
+from .phys_const import ge, muNbohr
+from .spin_utils import spinMat
+
+
+numThr = 1e-8
+maxDim = 1000 
+
+class spin:
```

Comment:
Classnames should start with a capital letter in order to make it easy to distinguish class constructions from function calls.

Reply:
Ah OK, I am learning :) ... can you apply the change?